src/pageServices/contentScripts/utils.ts

```diff
@@ -5,17 +5,37 @@ export const utilityScript = () => {
       const length = nodeSnapshot.snapshotLength;
       const foundElement = nodeSnapshot.snapshotItem(0) as Element;
       const foundHash = foundElement && foundElement.getAttribute("jdn-hash");
-      return JSON.stringify({ length, foundHash, element_id, foundElement: foundElement.outerHTML, originJdnHash });
+      const foundElementText = foundElement.textContent;
+      return JSON.stringify({ length, foundHash, element_id, foundElementText, originJdnHash });
     } catch (error) {
       return "The locator was not found on the page.";
     }
   };
 
-  const assignJdnHash = ({ xPath, jdnHash }: Record<string, string>) => {
+  const evaluateCssSelector = ({ selector, element_id, originJdnHash }: Record<string, string>) => {
     try {
-      const nodeSnapshot = document.evaluate(xPath, document, null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null);
-      const length = nodeSnapshot.snapshotLength;
-      const foundElement = nodeSnapshot.snapshotItem(0) as Element;
+      const foundElements = document.querySelectorAll(selector);
```

Comment:
let's use `querySelector`, since there's no need in `...All`

Reply:
fixed